server.js

```diff
@@ -2,10 +2,10 @@ const mongoose = require('mongoose')
 const express = require('express')
 const bodyParser = require('body-parser')
 
-const User = require('./models/User')
-const Package = require('./models/Package')
-const Iternary = require('./models/Iternary')
-const Category = require('./models/Category')
+const User = require('./models/User') // Just for testing purpose
+const Package = require('./models/Package') // Just for testing purpose
+const Iternary = require('./models/Itenary') // Just for testing purpose
```

Comment:
Any reason to why you didn't update imports here? @TheMightyRaider

Reply:
Because I had to modifiy the testing code in the bottom as well